assets/jsonsnippets.jsonc

```diff
@@ -0,0 +1,31 @@
+{
+    /*

```

Comment:
Can we fix the formatting here?

Reply:
There are some issues I'm going to need to go through.  I consider this just a initial checkin.

Reply:
I'd fix them now but I'd rather have an initial checkin that has exactly the previous snippets so we can track changes.